kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/formats/BytesFormatChunkedStreamReader.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Lenses.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.lenses.streamreactor.connect.aws.s3.formats
+
+import java.io.InputStream
+
+import io.lenses.streamreactor.connect.aws.s3.config.BytesWriteMode
+import io.lenses.streamreactor.connect.aws.s3.model.{BucketAndPath, ByteArraySourceData, BytesOutputRow}
+
+import scala.util.Try
+
+class BytesFormatChunkedStreamReader(inputStreamFn: () => InputStream, fileSizeFn: () => Long, bucketAndPath: BucketAndPath, bytesWriteMode: BytesWriteMode, chunkSizeBytes: Int) extends S3FormatStreamReader[ByteArraySourceData] {
```

Comment:
We cannot support this. If we split binary content in chunks there is no way it can be pieced back correctly

Reply:
Chunking has now been removed.